snapshot.yaml

```diff
@@ -35,12 +35,14 @@ packages:
     commit: 8a2432891e96a547777b67a67135e235a1dce80c
 
   # https://github.com/haskell-servant/servant-swagger/pull/125
-  - git: https://github.com/haskell-servant/servant-swagger
-    commit: 1909e44e965dca24cb1f5cee4b08c0781dfdbff6
+  - git: https://github.com/haskell-servant/servant
+    commit: c388c5e82c3df5d8ae63b523fea012c23e37410a
+    subdirs:
+      - servant-swagger
```

Comment:
The servant-swagger package moved into the top-level servant repository. See https://github.com/haskell-servant/servant-swagger/commit/7ac475a40b4b95a50bc2094f9461c62bd7fbb1fd and https://github.com/haskell-servant/servant/pull/1475.

Reply:
I had to go back to the other repo (https://github.com/EdutainmentLIVE/snapshot/commit/390bffe79a398be3fbb1e9914b6d4becc6dc09b9) because the build failed: https://itprotv.semaphoreci.com/jobs/2f2e7e4e-506d-495a-a74b-af75ea8dd55b#L10895

Reply:
I opened an issue for this: https://github.com/haskell-servant/servant/issues/1507